src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -660,7 +653,11 @@ private uint getMethodAttribsInternal(MethodDesc method)
             // TODO: Cache inlining hits
             // Check for an inlining directive.
 
-            if (method.IsNoInlining)
+            if (method.IsRawPInvoke())
+            {
+                result |= CorInfoFlag.CORINFO_FLG_FORCEINLINE;
```

Comment:
Do we really need to have `CORINFO_FLG_FORCEINLINE` in the mix? Why it is not enough to use `JIT_FLAG_IL_STUB` to get the desired behavior?

Reply:
We do not need it. I took a closer look at the code, and this flag won't be needed. I'm removing it.